composer.json

```diff
@@ -35,7 +35,8 @@
         "aura/intl": "^3.0.0",
         "psr/log": "^1.0.0",
         "psr/simple-cache": "^1.0.0",
-        "zendframework/zend-diactoros": "^1.4.0"
+        "zendframework/zend-diactoros": "^1.4.0",
+        "paragonie/random_compat": "^2.0|9.99.99"
```

Comment:
What does requiring 9.99.99 do?

Reply:
PHP5 will install v2.0 with `lib/random.php` autoloaded.

PHP7 will install v9.99.99 with no file autoloaded.

Reply:
I see now. I was looking in master branch.  Very nice, so this solves our problem.